src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-thu calendar--month-lenght-28">
```

Comment:
following BEM recommendation for naming classes
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-lenght--28">
```